### DISCUSSION TITLE
Would it be safe to upgrade Bash to 5.1?

### DISCUSSION THREAD

**Discussion by matteocoder** [git-for-windows/git#3880] [Q&A] [answered]
I noticed that Git for Windows comes with Bash 4.4. Since the MSYS2 package repository has Bash 5.1, would it introduce breakages if I downloaded the package from there and used it instead of the built-in `bash.exe`?

**Comment by dscho** [comment, marked as answer]
It has been announced via the release notes that we will do that "soon after v2.36.0". There are still known regressions in the v2.36.x release train, therefore it is still _too_ soon.

**Reply by matteocoder** [reply]
Thanks for your reply.
